app/helpers/admin_helper.rb

```diff
@@ -59,6 +59,10 @@ def trending_ips?(since: 1.hour.ago, limit: 10)
     !trending_ips(since: since, limit: limit).empty?
   end
 
+  def admin_site_settings
+    @admin_site_settings ||= Admin::Site.first_or_create!
+  end
+
```

Comment:
Remove this and add a `before_action` in `Admin::AdminController` to load the settings object